Formula/mesa.rb

```diff
@@ -23,13 +23,14 @@ class Mesa < Formula
   depends_on "linuxbrew/xorg/libvdpau" => :build
   depends_on "linuxbrew/xorg/wayland-protocols" => [:recommended, :build]
   depends_on "linuxbrew/xorg/xorgproto" => :build
-  depends_on "llvm@4" => :build
+  depends_on "llvm@7" => :build
   depends_on "pkg-config" => :build
-  depends_on "python" => :build
+  depends_on "python@2" => :build
   depends_on "libelf"
   depends_on "linuxbrew/xorg/libdrm"
   depends_on "linuxbrew/xorg/libomxil-bellagio"
   depends_on "linuxbrew/xorg/libxdamage"
+  depends_on "linuxbrew/xorg/libxrandr"
```

Comment:
why do we need it? Is it linked against in mesa 19? Could you please show `brew linkage mesa` when you build `mesa` locally?

Reply:
Thanks for tellng me about this command as it seems quite reliable compared to a manual check. The answer appears to be no. I'd have to recall the error that failed my build. I need to investigate the situation further.

Reply:
OK, `libxrandr` _is_ necessary to build this new version of `mesa`:
```
Last 15 lines from /home/linuxbrew/.cache/Homebrew/Logs/mesa/02.autogen.sh:
checking for x11-xcb xcb xcb-dri2 >= 1.8 xcb-xfixes... yes
checking for x11-xcb xcb >= 1.9.3 xcb-dri3 xcb-xfixes xcb-present xcb-sync xshmfence >= 1.1... yes
checking for xcb-dri3 >= 1.13 xcb-present >= 1.13... yes
checking for x11-xcb xcb-randr... yes
checking for xrandr... no
configure: error: Package requirements (xrandr) were not met:

No package 'xrandr' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables XLIB_RANDR_CFLAGS
and XLIB_RANDR_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
/usr/bin/curl --version
```
but `mesa` does not linked against `libxrandr`. This means that it should be a `:build` dependency unless some paths related to `libxrandr` are hard-coded into `mesa` files...

Reply:
`grep -r libxrandr *` in `mesa`'s cellar as well as grepping `libxrandr` from the output of `strings` returned nothing, so I think it is safe to set `libxrandr` as a `:build` dependency.

Reply:
Oh my, I did not notice your replies on this comment 😯 but good that we've come up with similar conclusions.

Reply:
no worries!